styles/style.less

```diff
@@ -201,7 +201,7 @@ div#scoreboardPlayer {
 				border-color: @color-border-hard;
 			}
 
-			#game.bets &:checked + .labelDie {
+			&:checked + .labelDie {
 				border-style: dashed;
```

Comment:
The idea was that dashed lines were a hint about what dice to notice when betting, while solid lines were a strong confirmation that the die was to be acted upon. when rerolling.

I'm slightly against this change tbh, I think it's actually more confusing treating all the cases the same. Do you think we could differentiate through other means, such as border colors? Or maybe dotted lines are more distinct than dashed vs solid and therefore you find the meaning clearer?

Reply:
The behaviour in `main` is:
* Solid when rolling.
* Dashed when betting.
* Solid when showing results.

Is that last part intended too?

Reply:
Then the fix is to set it as 4px dashed transparent by default, and changing it to solid in #game.rolling.